circleci/submodule-github-release

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Publishes content from [MODULE_PATH] as a Github Release.
+# Repo must have a VERSION file in its base directory.
+#
+# Usage:
+#
+#   github-release [--pre-release] <GITHUB_TOKEN> [MODULE_PATH] 
```

Comment:
Can you clarify that `MODULE_PATH` is the relative path to the submodule, not (for example) the module path as defined by https://go.dev/ref/mod#module-path ?